site/labs/gates/basic-gates.rst

```diff
@@ -0,0 +1,192 @@
+***********
+Basic Gates
+***********
+
+* Feel free to use your laptop
+* You are strongly encourage to work with others
+
+    * When you get stuck, ask those sitting around you for help
+    * Get used to working together in the labs
+    * Peer teaching and peer learning has been empirically shown to be very effective
+
+
+For these questions, feel free to use a single Digital workspace for all the circuits. However, label each circuit with
+labelled rectangles. These are components that can be found under "Components -> Misc. -> Decoration -> Rectangle".
+
+Where possible, each question should have complete tests. The test component can be found under "Components -> Misc. ->
+Test case". Once a test component is placed on the workspace, right click on the component to edit the test data.
+
+Questions may have restrictions on the logic gates that may be used. When restrictions are stated, they only apply to
+the gates; inputs, outputs, wires, etc. may be still be used.
+
+
+
+Creating Gates with Universal Operators
+=======================================
+
+For these questions, although only a single gate type may be used, feel free to use as many copies of these gates as
+needed.
+
+
+``NAND``
+--------
+
+#. Using only ``NAND``, create a circuit that performs ``NAND``
+
+    * This circuit is intentionally trivial
+
+
+#. Using only ``NAND``, create a circuit that performs ``NOT``
+#. Using only ``NAND`` and ``NOT``, perform ``AND``
+#. Using only ``NAND`` and ``NOT``, perform ``OR``
+#. Using only ``NAND`` and ``NOT``, perform ``NOR``
```

Comment:
Any reason to switch to `NOT` gates as well? Same for `NOR` below. I get they will have just proven that these gates can produce `NOT` as well, but it still feels odd to take a small step around the whole _universal_ness.

Reply:
The motivation is exactly what you said --- they built and proved it, so they "unlocked" it, so to speak. Like building a layer of abstraction. 

It could go either way to allow them to use `NOT` vs. _not_ allowing them, but the reasoning for the decision is based on the assignment making them do things a little more long form (for lack of a better term).